pyirf/spectral.py

```diff
@@ -242,6 +245,8 @@ class TableInterpolationSpectrum:
     By default, flux is interpolated linearly in log-log space.
     '''
     def __init__(self, energy, flux, log_energy=True, log_flux=True, reference_energy=1 * u.TeV):
+        '''Create a new TableInterpolationSpectrum spectrum'''
+        super().__init__(normalization=normalization, index=index, e_ref=e_ref)
```

Comment:
this does not seem to be related to the PR, and also the commit says only "Add docstring for __init__ methods ", while this last line does something different, was it intended ?

Reply:
Good catch, I unintentionally copied two lines.